### ISSUE TITLE
Install detected Flutter version directly

### ISSUE THREAD

**Issue by spydon** [kuhnroyal/flutter-fvm-config-action#24]
An enhancement proposal that would simplify it a lot for the users of this actions is to install the set fvm flutter version directly, instead of just reading out the config.
Similarly to how asdf does it: https://github.com/asdf-vm/actions

**Comment by spydon** [comment]
Found another action that does this here (but with non existing docs): https://github.com/Atsumi3/actions-setup-fvm

**Comment by kuhnroyal** [comment]
Hey Lukas, great idea, never thought of it this way.
Do you think defaulting to this behavior in a new major version is fine? :)

**Comment by spydon** [comment]
> Hey Lukas, great idea, never thought of it this way.
> Do you think defaulting to this behavior in a new major version is fine? :)

For sure! That's what major versions are there for 😃

**Comment by kuhnroyal** [comment]
What are your thoughts about a new input vs separate actions?

For now I havethis.
```yaml
      - uses: kuhnroyal/flutter-fvm-config-action@v2
        with:
          # The setup flag will default to true in the next major version (v3)
          setup: true
```

**Comment by spydon** [comment]
A well explained input sounds like a good idea :)

**Comment by kuhnroyal** [comment]
This is now implemented in v2.3 via an input and in v3 additionally via 2 separate sub actions.
